### DISCUSSION TITLE
v 1.1.1

### DISCUSSION THREAD

**Discussion by ghost** [kkebo/DNSecure#32] [General]
Dear Kubo-san
the 1.1.1 release is working fine.

I have only one suggestion. Would be nice to replace:
DNS-over-HTTPS with **DoH** and DNS-over-TLS with **DoT**

**Comment by kkebo** [comment]
Hi @zero0ID,

Thanks for suggestion. Could you tell me why you think it's better?

**Reply by ghost** [reply]
Ha, just because is much shorter and "looks" better.

**Reply by kkebo** [reply]
Thank you. I'll consider it, but I don't feel the necessity of changing that at this time. I believe that DNS-over-* is easier to understand than Do*. If I remember correctly, Cloudflare's 1.1.1.1 app used the words without abbreviations as well.

**Comment by ghost** [comment]
OK,
it was just an idea, maybe I like too much the simplicity :-)

-0ID


8 Jan 2021, 19:01 by notifications@github.com:

**Reply by kkebo** [reply]
I understand what you are saying. Depending on a location of label, I may adopt shorter ones that you suggested. Thanks.

**Comment by ghost** [comment]
Here is the reason to change the description.

![DoH](https://user-images.githubusercontent.com/76554148/104067035-b54bfb80-5202-11eb-8b5d-fec74a1a6bde.jpg)

**Reply by kkebo** [reply]
@zero0ID I thought you were talking about the add menu, but in that case, abbreviated forms are indeed better. I'll work on it. Thanks.
